src/transformers/feature_extraction_utils.py

```diff
@@ -469,6 +470,16 @@ def get_feature_extractor_dict(
                 f"loading configuration file {feature_extractor_file} from cache at {resolved_feature_extractor_file}"
             )
 
+        if "auto_map" in feature_extractor_dict and not is_local:
+            feature_extractor_dict["auto_map"] = {
+                k: (f"{pretrained_model_name_or_path}--{v}" if "--" not in v else v)
+                for k, v in feature_extractor_dict["auto_map"].items()
+            }
```

Comment:
Is this needed before the block below?

Reply:
Woops, no. Just a bad copy/paste thanks for catching!